resources/aws/route.go

```diff
@@ -1,14 +1,15 @@
 package aws
 
 import (
+	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/giantswarm/microerror"
 )
 
 type Route struct {
-	DestinationCidrBlock string
-	VpcID                string
-	RouteTable           RouteTable
+	DestinationCidrBlock   string
+	VpcPeeringConnectionID string
+	RouteTable             RouteTable
```

Comment:
This is some boyscouting to rename the VPC peering ID. As it confused me when I was adding the new routes.